PWGDQ/Tasks/tableReader_withAssoc.cxx

```diff
@@ -1661,21 +1708,31 @@ struct AnalysisAsymmetricPairing {
               names.push_back(Form("%s_%s_%s", pairHistPrefixes[iPrefix].Data(), legsStr.Data(), objArrayPair->At(iPairCut)->GetName()));
               histNames += Form("%s;", names[iPrefix].Data());
             }
-            fTrackHistNames[fNLegCuts + icut * fNPairCuts + iPairCut] = names;
+            fTrackHistNames[fNLegCuts * (fNCommonTrackCuts + 1) + icut * fNPairCuts + iPairCut] = names;
+            for (int iCommonCut = 0; iCommonCut < fNCommonTrackCuts; ++iCommonCut) {
+              names = {};
+              for (int iPrefix = 0; iPrefix < fNPairHistPrefixes; ++iPrefix) {
+                names.push_back(Form("%s_%s_%s_%s", pairHistPrefixes[iPrefix].Data(), legsStr.Data(), objArrayCommon->At(iCommonCut)->GetName(), objArrayPair->At(iPairCut)->GetName()));
+                histNames += Form("%s;", names[iPrefix].Data());
+              }
+              fTrackHistNames[(fNLegCuts * (fNCommonTrackCuts + 1) + fNLegCuts * fNPairCuts) + icut * (fNPairCuts * fNCommonTrackCuts + 1) + iCommonCut * (1 + fNPairCuts) + iPairCut] = names;
+            } // end loop (common cuts)
           } // end loop (pair cuts)
         }   // end if (pair cuts)
       }
     }
+    for (auto& pair : fTrackHistNames) {
```

Comment:
@sigurdnese  Remove this for loop since it is just a control printout

Reply:
My bad, thanks for spotting it!